app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container d-flex flex-column justify-content-between vh-100 bg-light px-0">
+  <div class="flex-1 m-0">
+    <div class="d-flex justify-content-between align-items-center header">
+      <%= link_to categories_path do %>
+        <span class="bi bi-arrow-left" style="font-size:24px;"></span>
+      <% end %>
+      <span style="font-size:20px;">TRANSACTIONS</span>
+      <%= button_to "", class: "btn btn-small btn-outline text-white" do %>
+        <i class="bi bi-search"></i>
+      <% end %>
+    </div>
+    <div class="d-flex justify-content-between align-items-center shadow-sm bg-white w-100 p-2">
+      <span class="align-middle"><%= @category.name %></span>
+      <span class="align-middle"><%= number_to_currency(@category.total_amount) %></span>
+    </div>
+    <ul class="list-group rounded-0 px-3 mt-3">
+      <% @expenses.each do |expense| %>
+        <li class="list-group-item shadow-sm justify-content-between align-items-center mb-3">
+          <div class="d-flex justify-content-between align-items-center w-100 py-1 gap-2">
+            <span class="flex-1 align-middle"><%= expense.name %></span>
+            <span class="align-middle"><%= number_to_currency(expense.amount) %></span>
+          </div>
+        </li>
+      <% end %>
```

Comment:
- [ ] As per requirements, please modify the controller index method to return here in the views, the expenses sorted by the most recent ones. 🙏

Reply:
well resolved! ✔️